core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -94,6 +95,249 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     super.tearDown()
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testAclOperationsWithOptionTimeoutMs(): Unit = {
+    val acl = new AclBinding(new ResourcePattern(ResourceType.TOPIC, "mytopic3", PatternType.LITERAL),
+      new AccessControlEntry("User:ANONYMOUS", "*", AclOperation.DESCRIBE, AclPermissionType.ALLOW))
+    client = createAdminClient
+    val startTimeMs = Time.SYSTEM.milliseconds()
+    assertFutureExceptionTypeEquals(client.createAcls(Collections.singleton(acl), new CreateAclsOptions().timeoutMs(2)).all(), classOf[TimeoutException])
```

Comment:
this is unstable since it may pass when the test run fast enough. Maybe we can create a `Admin` with incorrect bootstrap and then send this request with timeout.